src/app/app-routing.module.ts

```diff
@@ -1,10 +1,25 @@
-import { NgModule } from '@angular/core';
-import { RouterModule, Routes } from '@angular/router';
+import {NgModule} from '@angular/core';
+import {RouterModule, Routes} from '@angular/router';
 
-const routes: Routes = [];
+import {AddMovieComponent} from "./components/add-movie/add-movie.component";
+import {WrapperComponent} from "./components/wrapper/wrapper.component";
+
+const routes: Routes = [
+  {
+    path: '',
+    component: WrapperComponent,
```

Comment:
WrapperComponent is a bad name, I would add MoviesPageComponent, but it's up to you

Reply:
I'm agree. In previous branch I have already renamed it into HomeComponent